frameworks/Scala/finch/src/main/scala/Main.scala

```diff
@@ -0,0 +1,47 @@
+import java.text.{DateFormat, SimpleDateFormat}
+import java.util.Date
+
+import com.twitter.io.Buf
+import com.twitter.finagle.http.Response
+import com.twitter.finagle.Http
+import com.twitter.finagle.stats.NullStatsReceiver
+import com.twitter.finagle.tracing.NullTracer
+import com.twitter.util.Await
+
+import io.circe.Json
+import io.finch._
+import io.finch.circe._
+
+object Main extends App {
+
+  val dateFormat: ThreadLocal[DateFormat] = new ThreadLocal[DateFormat] {
+    override def initialValue: DateFormat = new SimpleDateFormat("E, dd MMM yyyy HH:mm:ss z")
```

Comment:
Why don't you use Java 8 DateTimeFormatter instead ?
It is thread-safe, so you'll need just one instance.